Dockerfile

```diff
@@ -82,16 +88,14 @@ ENV SHELLCHECK_VERSION=0.4.3
 RUN curl -L https://s3-us-west-2.amazonaws.com/get-deis/shellcheck-$SHELLCHECK_VERSION-linux-amd64 -o /usr/local/bin/shellcheck \
     && chmod +x /usr/local/bin/shellcheck
 
-# copy everything to rootfs
-COPY rootfs /
-
 # change ownership of everything in $JENKINS_HOME to jenkins
 RUN chown -R jenkins:jenkins $JENKINS_HOME
 
+COPY rootfs/bin/jenkins-slave /usr/local/bin/jenkins-slave
```

Comment:
How about we keep this as `COPY rootfs /` and dump the jenkins-slave script in `rootfs/usr/local/bin`?